build.sbt

```diff
@@ -10,6 +10,26 @@ lazy val root = (project in file("."))
   )
 
 libraryDependencies ++= Seq(
-  awsSdkDynamoDb,
-  scalaTest % Test
+  awsSdkDynamoDbV2,
```

Comment:
The dependencies that are used only for the testing should be annotated with `%Test`.

This means that are not pulled in for actual Lambda code when its built, making the code more "light weight".

- `awsSdkDynamoDbV2`
- `awsSdkDynamoDbV1`
- `testContainer`
- `testContainerDynalite`

Reply:
sensible. I've done that now.